app/models/manageiq/providers/redhat/infra_manager/vm_import.rb

```diff
@@ -60,9 +84,14 @@ def perform_vmware_to_ovirt_import(params)
           :sparse         => params[:sparse],
         )
       )
+      ems_ref_for_vm_id(import.vm.id)
     end
   end
 
+  def ems_ref_for_vm_id(vm_id)
+    "/api/vms/#{vm_id}"
```

Comment:
@matobet Is this how RHEVM's represents the ems_ref ids and how we store in our database? Does this string match how the ems_refresh is going to build the vm information